image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl

```diff
@@ -193,13 +153,20 @@
 
 {{/* Provide some human-readable feedback regarding the Scanner V4 configuration to the user installing the Helm chart. */}}
 {{- if and (get $components "indexer") (not (get $components "matcher")) -}}
```

Comment:
Below we are changing the human-readable output for the NOTES.txt to be prettier.